modules/view_establishment/module.py

```diff
@@ -205,6 +205,33 @@ def send_msg(self):
                               self.witnesses[node_j],
                               self.pred_and_action.get_info(node_j)
                               ]
+
+                # Overwriting own_data to send different views to different
+                # nodes, to trick them
+                # if acting Byzantine with different_views - behaviour
+                if (byz.is_byzantine() and
```

Comment:
Nit: Might make sense to do 

```
if byz.is_byzantine():
    if (byz.get_behavior() == byz.DIFFERENT_VIEWS:
        ...
    elif byz.get_behavior() == byz.FORCING_RESET:
        ...
```

this indicates that only one byz behavior is valid at a given point in time. not a big deal, just makes it a bit clearer when reading the code

Reply:
Of course!